sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -165,6 +168,26 @@ def __init__(
                 cloud_name,
             )
         module_logger.debug("Cloud configured in MLClient: '%s'.", cloud_name)
+
+        # registry_name is present when the operations need referring assets from registry.
+        # the subscription, resource group, if provided, will be ignored and replaced by
+        # whatever is received from the registry discovery service.
+        if registry_name:
```

Comment:
Do we plan to add the ws_name and registry_name mutual exclusiveness check in a separate PR?

Reply:
Yes, but that will come on a different PR, aiming to the 10/3 cut off. This PR is about allowing to perform operations in the SDK on registry w/o specifying the RG.

Reply:
@luigiw: Resolving the conversation since the same code was approved by you on ADO PR. I have answered the question, and we want to make sure this PR is in the release candidate with cutoff today.